model-engine/model_engine_server/inference/tensorrt-llm/triton_model_repo/postprocessing/1/model.py

```diff
@@ -147,10 +176,26 @@ def finalize(self):
         """
         print("Cleaning up...")
 
-    def _postprocessing(self, tokens_batch):
+    def _postprocessing(self, tokens_batch, sequence_lengths):
         outputs = []
-        for beam_tokens in tokens_batch:
-            for tokens in beam_tokens:
-                output = self.tokenizer.decode(tokens)
+        for batch_idx, beam_tokens in enumerate(tokens_batch):
+            for beam_idx, tokens in enumerate(beam_tokens):
+                seq_len = sequence_lengths[batch_idx][beam_idx]
+                output = self.tokenizer.decode(
```

Comment:
should we check for stop token here?

Reply:
I don't remember seeing stop tokens in the output when testing at least.